src/spikeinterface/core/sortinganalyzer.py

```diff
@@ -1930,19 +1990,33 @@ def merge(
 
     def run(self, save=True, **kwargs):
         if save and not self.sorting_analyzer.is_read_only():
-            # this also reset the folder or zarr group
+            # NB: this call to _save_params() also resets the folder or zarr group
             self._save_params()
             self._save_importing_provenance()
+            self._save_run_info()
 
+        start = time()
```

Comment:
Can you use `time.perf_counter()` instead ?

Reply:
All set